suite-native/navigation/src/components/ScreenHeader.tsx

```diff
@@ -1,48 +1,31 @@
 import React from 'react';
-import { TouchableOpacity } from 'react-native';
 
 import { useNavigation } from '@react-navigation/native';
 
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box, Text } from '@suite-native/atoms';
-import { Icon } from '@trezor/icons';
+import { IconButton, Text } from '@suite-native/atoms';
+
+import { ScreenHeaderWithIcons } from './ScreenHeaderWithIcons';
 
 type ScreenHeaderProps = {
     title?: string;
 };
 
-const GO_BACK_BUTTON_SIZE = 40;
-const iconStyle = prepareNativeStyle(utils => ({
-    backgroundColor: utils.colors.gray300,
-    height: GO_BACK_BUTTON_SIZE,
-    width: GO_BACK_BUTTON_SIZE,
-    borderRadius: utils.borders.radii.round,
-    justifyContent: 'center',
-    alignItems: 'center',
-}));
-
-const headerStyle = prepareNativeStyle(_ => ({
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-    paddingBottom: 10,
-    paddingTop: 15,
-}));
-
 export const ScreenHeader = ({ title }: ScreenHeaderProps) => {
```

Comment:
This component seems quite redundant now, we I don't see reason why we should have both `ScreenHeader` and `ScreenHeaderWithIcon`. Please delete this one and rename `ScreenHeaderWithIcon` to just `ScreenHeader`